ffmpeg/ffmpeg.go

```diff
@@ -49,9 +50,10 @@ type Transcoder struct {
 }
 
 type TranscodeOptionsIn struct {
-	Fname  string
-	Accel  Acceleration
-	Device string
+	Fname     string
+	Accel     Acceleration
+	Device    string
+	Transmuxe bool
```

Comment:
nit: can we rename this to `Transmux` or `Transmuxing` - never heard `Transmuxe` used anywhere.

Reply:
fixed